src/main/java/com/wrapper/spotify/model_objects/miscellaneous/AudioAnalysisSection.java

```diff
@@ -290,9 +290,10 @@ public AudioAnalysisSection createModelObject(JsonObject jsonObject) {
                               ? jsonObject.get("loudness").getAsFloat()
                               : null)
               .setMeasure(
-                      hasAndNotNull(jsonObject, "measure")
-                              ? new AudioAnalysisMeasure.JsonUtil().createModelObject(
-                              jsonObject.getAsJsonObject("measure"))
+                      (hasAndNotNull(jsonObject, "start")
+                              && hasAndNotNull(jsonObject, "duration")
+                              && hasAndNotNull(jsonObject, "confidence"))
```

Comment:
Those checks are not needed, are they? They are already part of the measure builder.

Reply:
Yeah that's my bad. Removing those checks would be better.